packages/cli-lib/lib/__tests__/uploadFolder.js

```diff
@@ -49,7 +49,7 @@ describe('uploadFolder', () => {
 
       uploadedFilesInOrder.forEach((file, index) => {
         expect(upload).nthCalledWith(index + 1, accountId, file, file, {
-          qs: { buffer: false },
+          qs: { buffer: false, environmentId: 1 },
```

Comment:
Will the `environmentId` field be ignored in accounts that do not have Projects access?

Reply:
It will default to `1` if the `--staging` option is not tacked onto the command.